web/reports.jsp

```diff
@@ -437,7 +437,7 @@ if (request.getParameter("projects") != null) {
                     for (int i=0;i<cntProjects;i++) {
                         String lastLogEntry = allProjects[i].getProjectLog(1);
                         if (lastLogEntry.length() == 0) lastLogEntry = "none recorded";
-                        int lastFolioID = allProjects[i].getLastModifiedFolio();
+                        int lastFolioID = allProjects[i].getLastModifiedFolio(true);
```

Comment:
I assume we're overriding the one that exists, but I am not sure why we are not just using it.